src/migrations/1743831360133-paginatedVCs.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class PaginatedVCs1743831360133 implements MigrationInterface {
+  name = 'PaginatedVCs1743831360133';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE \`virtual_contributor\` ADD \`rowId\` int NOT NULL AUTO_INCREMENT`
```

Comment:
interesting....  the error suggests there is already an `auto_increment` column and a second of the kind needs to have an index applied, which I don't believe is true?
Combining both statements will let SQL know that you are planning on applying an index to it.
```suggestion
      `ALTER TABLE \`virtual_contributor\` ADD \`rowId\` int NOT NULL AUTO_INCREMENT,
      ADD UNIQUE INDEX `IDX_a643bc875218dd4abbf86bbf7f` (`rowId`)`
```